Controller/NotifyHandler.php

```diff
@@ -169,4 +177,20 @@ private function _getParentTransactionId(Response $oResponse)
 
         return $sParentTransactionId;
     }
+
+    /**
+     * Returns the URL of the notification handler
+     *
+     * @param PaymentMethod $oPaymentMethod
+     *
+     * @return string
+     */
+    public static function getNotificationUrl($oPaymentMethod)
```

Comment:
Where is this used? Couldn't find it.

Reply:
Thanks, it's now used by the PaymentGateway class